experimental/packages/otlp-transformer/package.json

```diff
@@ -6,10 +6,48 @@
   },
   "version": "0.56.0",
   "description": "Transform OpenTelemetry SDK data into OTLP",
-  "module": "build/esm/index.js",
-  "esnext": "build/esnext/index.js",
-  "types": "build/src/index.d.ts",
-  "main": "build/src/index.js",
+  "exports": {
```

Comment:
Given things are conditionally exported now, do we also want to expose a `./generated` export so that projects that want to access the deserialization logic this package uses, they can just use this package for that?

Reply:
@SebastienGllmt I am pretty sure @pichlermarc have slightly different plans around the details, so I'd follow #5264 and #5263 instead; leaving this open for now but I'm pretty sure those combined would supersede this one